### PR TITLE
Improve test  coverage and small doc fixes

### PR DESCRIPTION
**Pull request by prescod**

### CODE REVIEW

File: snowfakery/data_gen_exceptions.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ def __init__(self, message, filename=None, line_num=None):
     def __str__(self):
         if self.line_num:
             location = f"\n near {self.filename}:{self.line_num}"
-        elif self.filename:
+        elif self.filename:  # pragma: no cover
```
Comment (davisagli) [inline snowfakery/data_gen_exceptions.py:18]: It looks easy to test; why not test it?

File: snowfakery/object_rows.py [hunk 2]
```diff
@@ -104,5 +100,5 @@ def status(self):
         elif self.allocated_id == SlotState.CONSUMED:
             return SlotState.CONSUMED
 
-    def __repr__(self):
+    def __repr__(self):  # pragma: no cover
```
Comment (davisagli) [inline snowfakery/object_rows.py:103]: Why not test it?

**Review by davisagli — APPROVED** [review]
Approving, but I'd prefer to reserve "no cover" for things that are hard to test.

File: snowfakery/data_gen_exceptions.py (hunk 1 above)
Comment (prescod) [reply snowfakery/data_gen_exceptions.py]: 2a90f0ce9

File: snowfakery/object_rows.py (hunk 2 above)
Comment (prescod) [reply snowfakery/object_rows.py]: 2a90f0ce9

**Comment by davisagli** [comment]
@prescod the builds are red

**Review by davisagli — APPROVED** [review]

**Pull request merged**